src/background.ts

```diff
@@ -52,6 +54,14 @@ const store = new Store<{
     fileEncoding: {
       type: "string",
     },
+    simpleMode: {
+      type: "object",
+      properties: {
+        enabled: { type: "boolean" },
+        avoid: { type: "boolean" },
```

Comment:
「上書きの防止」であることがわかる名前にすると良いと思います。

それと、この機能は書き出し先が固定されていなくても使えると便利だと思います。
なので、`simpleMode`の下ではなく、`simpleMode`の隣に並べるのはどうでしょう。

---

It would be nice to have a name that makes it clear that it is "Prevent Overwrite".

Also, I think it would be useful to use this feature even if the destination is not fixed.
So, how about placing it next to `simpleMode` instead of under `simpleMode`?